src/core-layers/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -152,7 +152,8 @@ export default class SolidPolygonLayer extends Layer {
     return new Model(gl, Object.assign({}, this.getShaders(), {
       id: this.props.id,
       geometry: new Geometry({
-        drawMode: this.props.wireframe ? GL.LINES : GL.TRIANGLES
+        drawMode: this.props.wireframe ? GL.LINES : GL.TRIANGLES,
+        attributes: {}
```

Comment:
@1chandu - Nit: this should not be needed. We should have a default value for attributes.